### PR TITLE
AI-106: Fix contact record top header issues

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
**Before**

align

hide-btn-and-padding

icon-restore

lable

lable2

radio

**After**

resloved-hide-btn-and-padding

resolved-align

resolved-icon-restore

resolved-lable

resolved-lable2

resolved-radio

### CODE REVIEW

**Comment by meladawy** [comment]
Please resolve the conflict

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_detail.scss [hunk 1]
```diff
@@ -473,6 +508,50 @@
...
+      input[type=radio] {
+        &:checked + label[for]:after {
+          top: 7px;
+        } 
+      }
+    }   
+  }
+}
+
+.contactCardRight {
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_detail.scss]: This is a 4 level hierarchy, try not to exceed 3. You can use it like this 
.contactCardRight #customData {

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_detail.scss [hunk 2]
```diff
@@ -473,6 +508,50 @@
...
+    }
+
+    &#Demographics {
+      input.dateplugin {
+        border-color: $crm-darker-grayblue;
+        margin-top: 0;
+      }      
+    } 
+
+    &#CustomData {
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_detail.scss]: This is exceeding 4 level hierarchy plz try to  minimize this...maybe through
&#CustomData input[type=radio] {

Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_detail.scss]: Fixed.

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_detail.scss (hunk 1 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_detail.scss]: Fixed.

**Pull request merged**